File: app/src/main/java/com/chooloo/www/callmanager/ui/activity/settings/SettingsActivity.java
package com.chooloo.www.callmanager.ui.activity.settings;

import android.os.Bundle;

import androidx.fragment.app.Fragment;

import com.chooloo.www.callmanager.R;
import com.chooloo.www.callmanager.ui.activity.base.BaseThemeActivity;
import com.chooloo.www.callmanager.util.ThemeUtils;

import java.util.Objects;

public class SettingsActivity extends BaseThemeActivity implements SettingsContract.View {
    private static final String TAG_FRAGMENT = "fragment";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setThemeType(ThemeUtils.TYPE_NORMAL);
        setContentView(R.layout.activity_settings);
    }

    @Override
    public void setUp() {
        Fragment fragment = new SettingsFragment();
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.fragment, fragment, TAG_FRAGMENT)
                .commit();

        Objects.requireNonNull(getSupportActionBar()).setDisplayHomeAsUpEnabled(true);
    }
}

File: app/src/main/java/com/chooloo/www/koler/ui/settings/SettingsActivity.kt
package com.chooloo.www.koler.ui.settings

import android.os.Bundle
import com.chooloo.www.koler.databinding.ActivitySettingsBinding
import com.chooloo.www.koler.ui.base.BaseActivity
import com.chooloo.www.koler.ui.settingsfragment.SettingsFragment

class SettingsActivity : BaseActivity(), SettingsContract.View {
    private val _binding by lazy { ActivitySettingsBinding.inflate(layoutInflater) }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(_binding.root)
    }

    override fun onSetup() {
        supportFragmentManager
            .beginTransaction()
            .add(_binding.settingsFragmentContainer.id, SettingsFragment.newInstance())
            .commitNow()
    }
}